app/src/main/java/com/example/voyagerx/repository/model/Launch.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.voyagerx.repository.model
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
+data class Launch(
+    @PrimaryKey
+    val id: String,
+
+    val mission_name: String,
+    val launch_site_long: String,
+    val launch_date_utc: String,
+    val details: String,
+
+    val article_link: String,
+    val video_link: String,
+    val image_links: List<String>
```

Comment:
Are these fields nullable? I'm not sure too sure about Android Room syntax.

I think these should be nullable in case a launch has a null value in a certain field.

Reply:
Same idea. If there's a chance these values can come back from the API as null or empty, then we should reflect that here as nullable with the Kotlin '?' syntax: https://kotlinlang.org/docs/null-safety.html